app/models/scoreboard.rb

```diff
@@ -67,4 +67,10 @@ def scored_schools(recent_boundary: 1.month.ago, academic_year: nil)
   def position(school)
     scored_schools.index(school)
   end
+
+  private
+
+  def this_academic_year
```

Comment:
Should probably just force this to be passed in at all times so we're always using the correct academic year.